examples/clusterimportpolicy/karmada.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: policy.clusterpedia.io/v1alpha1
+kind: ClusterImportPolicy
+metadata:
+  name: karmada
+spec:
+  source:
+    group: "cluster.karmada.io"
+    versions: []
+    resource: clusters
+  references:
+    - group: ""
+      resource: secrets
+      namespaceTemplate: "{{ .source.spec.secretRef.namespace }}"
+      nameTemplate: "{{ .source.spec.secretRef.name }}"
+      key: secret
+  nameTemplate: "karmada-{{ .source.metadata.name }}"
+  template: |
+    spec:
+      apiserver: "{{ .source.spec.apiEndpoint }}"
+      tokenData: "{{ .references.secret.data.token }}"
+      caData: "{{ .references.secret.data.caBundle }}"
+      syncResources:
+        - group: ""
+          resources:
+            - "pods"
```

Comment:
We could synchronize some common resources by default.
```yaml
      syncResources:
        - group: ""
          resources:
            - "pods"
            - "services"
        - group: "apps"
          resources:
            - "*"
```

Reply:
Yes, it is necessary